docs/tutorials/model_validation.md

```diff
@@ -233,9 +229,9 @@ the most common use case, so this is treated specially in BuildingMOTIF.
 
 ```{code-cell}
 # load manifest into BuildingMOTIF as its own library!
-manifest = Library.load(ontology_graph="tutorial2_manifest.ttl")
+manifest = Library.load(ontology_graph="tutorial1_manifest.ttl")
 # set it as the manifest for the model
-model.update_manifest(manifest)
+model.update_manifest(manifest.get_shape_collection())
```

Comment:
Fix for https://buildingmotif.readthedocs.io/en/fix-rtd-deps/tutorials/model_validation.html#adding-the-manifest-to-the-model.

Reply:
Yes!